### DISCUSSION TITLE
zabbix with infinity

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#702] [General]
Hello,

in v1.3.0 making a zabbix api call works seemlessly with the latest version 2.1.0 , I am getting the following error

![image](https://github.com/yesoreyeram/grafana-infinity-datasource/assets/900951/187d6c34-ab09-487b-9703-41937aee7567)

This is the api endpoint for zabbix:

http://localhost:8084/api_jsonrpc.php

What is this Allowed Hosts ? Cant seem to find it in documentation?

Thanks

**Comment by yesoreyeram** [comment]
In the config authentication/security you have to add `http://localhost:8084` Read more https://sriramajeyam.com/grafana-infinity-datasource/docs/url#allowed-hosts

**Reply by yosiasz** [reply]
that does it thanks a lot!

**Reply by yesoreyeram** [reply]
Glad that worked for you. BTW, I strongly recommend to use native zabbix plugin where possible. If that's not working for you, create an issue with that repo. Actually Zabbix is very excellent plugin.

**Reply by yosiasz** [reply]
tried it but was not able to configure it hence to good old reliable infinity
